docs/Project.toml

```diff
@@ -11,4 +11,5 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
-Documenter = "0.27"
+Documenter = "1"
+julia = "1.10" # For jldoctest
```

Comment:
Do you know why this is required? Do you know how to use the latest stable release? (so that we do not have to update this file with every Julia release?)

Reply:
>Do you know why this is required? Do you know how to use the latest stable release?

Actually, I don't know. Perhaps caches are related and adding the julia compat resolved the version, I guess.

>so that we do not have to update this file with every Julia release?)

This compat also accepts v1.11 and higher, so we don't have to update with every Julia release.
This entry should be updated when REPL outputs are changed.